webapp/src/ts/components/actionbar/actionbar.component.html

```diff
@@ -61,58 +66,68 @@
       </div>
 
       <div class="col-sm-4 general-actions left-pane" *ngIf="currentTab === 'contacts'">
-        <div test-id='create_places' class="actions dropup" mmAuth [mmAuthAny]="[ actionBar?.left?.childPlaces && 'can_create_places' ]">
-          <a *ngIf="actionBar?.left?.childPlaces?.length > 1"
-             mmAuth="can_create_places"
-             class="mm-icon mm-icon-inverse mm-icon-caption dropdown-toggle"
-             data-toggle="dropdown"
-          >
-            <span class="fa fa-plus"></span>
-            <p>{{'Add place' | translate}}</p>
-          </a>
+        <div mmAuth [mmAuthAny]="[ 'can_edit', 'can_export_all', 'can_export_contacts' ]">
```

Comment:
Same here.

Reply:
Export might not be dependent on `can_edit`